packages/office-ui-fabric-react/src/components/pickers/examples/TagPicker.Basic.Example.tsx

```diff
@@ -23,7 +23,8 @@ const _testTags: ITag[] = [
   'rose',
   'violet',
   'white',
-  'yellow'
+  'yellow',
+  'あいうえお'
```

Comment:
DId you mean to add this here?

Reply:
Should have read your comment. Ignore this.

Reply:
This was to have a deploy that's testable, I will delete it before merging! Hence the 'Do not merge' label on the PR now